docs/sdks/android/GETTING_STARTED.md

```diff
@@ -72,8 +74,10 @@ val client = Client(context)
 val account = Account(client)
 val user = account.create(
     ID.unique(),
```

Comment:
Is there a reason why Kotlin was switched to named params instead of what's used before?

Reply:
Named params is better I thought.
Same I made earlier in Python with named params, so made this to look like consistent in other files as well. 
So shall I keep it unnamed params then @gewenyu99 ?